javascript/javascript2/week3/fastestGame/script.js

```diff
@@ -0,0 +1,40 @@
+console.log('Just test');
+function finishGame() {
+  let myBtn = document.getElementById('gameBtn');
+  myBtn.addEventListener('click', () => {
+    const time = document.getElementById('textInput').value;
+    setTimeout(() => {
+      const text = document.getElementById('overText');
+      text.innerHTML = 'Game over!!!';
+      if (keyboardS < keyboardL) {
+        alert('LLLLL winnnn !!!');
```

Comment:
Instead of alerting the user, you can also change the HTML text, it makes more sense to not use alert here!